tests/cupy_tests/core_tests/test_ndarray_complex_ops.py

```diff
@@ -72,3 +74,18 @@ def test_imag_setter_raise(self, xp, dtype):
         x = testing.shaped_arange((2, 3), xp, dtype)
         x.imag = testing.shaped_reverse_arange((2, 3), xp, dtype)
         return x
+
+
+@testing.gpu
+class TestScalarConversion(unittest.TestCase):
+
+    _multiprocess_can_split_ = True
+
+    @testing.for_all_dtypes()
+    def test_scalar_conversion(self, dtype):
```

Comment:
Cloud you add `@testing.with_requires('numpy>=1.12.0')`?
That error caused by old NumPy (<1.12.0).

Reply:
Ah right, Numpy has no `__complex__` before 1.12. I'll fix it.

Reply:
Done

Reply:
Thanks